elytron/src/test/resources/org/wildfly/extension/elytron/filesystem-realm-2/s/e/secondUser.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<identity xmlns="urn:elytron:1.0">
```

Comment:
RealmsTestCase.testAggregateRealmWithPrincipalTransformer uses attributes from this file, but values are the same as values in firstUser.xml. So basically test doesn't prove that values from secondUser.xml are really used (and doesn't prove that principal is really used). WDYT about these updates?

```diff
diff --git a/elytron/src/test/java/org/wildfly/extension/elytron/RealmsTestCase.java b/elytron/src/test/java/org/wildfly/extension/elytron/RealmsTestCase.java
index 2f4eb479d9..4816259228 100644
--- a/elytron/src/test/java/org/wildfly/extension/elytron/RealmsTestCase.java
+++ b/elytron/src/test/java/org/wildfly/extension/elytron/RealmsTestCase.java
@@ -182,9 +182,9 @@ public class RealmsTestCase extends AbstractSubsystemBaseTest {
         Assert.assertTrue(identity1.exists());
         //Assert that transformation was successful and the correct identity and attributes were loaded from filesystem-realm-2
         Assert.assertEquals(3, identity1.getAuthorizationIdentity().getAttributes().size());
-        Assert.assertEquals("[Jane]", identity1.getAuthorizationIdentity().getAttributes().get("firstName").toString());
-        Assert.assertEquals("[Doe]", identity1.getAuthorizationIdentity().getAttributes().get("lastName").toString());
-        Assert.assertEquals("[Employee, Manager, Admin]", identity1.getAuthorizationIdentity().getAttributes().get("roles").toString());
+        Assert.assertEquals("[Jane2]", identity1.getAuthorizationIdentity().getAttributes().get("firstName").toString());
+        Assert.assertEquals("[Doe2]", identity1.getAuthorizationIdentity().getAttributes().get("lastName").toString());
+        Assert.assertEquals("[Employee2, Manager2, Admin2]", identity1.getAuthorizationIdentity().getAttributes().get("roles").toString());
 
         identity1.dispose();
     }
diff --git a/elytron/src/test/resources/org/wildfly/extension/elytron/filesystem-realm-2/s/e/secondUser.xml b/elytron/src/test/resources/org/wildfly/extension/elytron/filesystem-realm-2/s/e/secondUser.xml
index 3048c2c192..45a6b00d8a 100644
--- a/elytron/src/test/resources/org/wildfly/extension/elytron/filesystem-realm-2/s/e/secondUser.xml
+++ b/elytron/src/test/resources/org/wildfly/extension/elytron/filesystem-realm-2/s/e/secondUser.xml
@@ -2,10 +2,10 @@
 <identity xmlns="urn:elytron:1.0">
     <!-- generator: FileSystemSecurityRealmTest.testCreateIdentityWithEverything() -->
     <attributes>
-        <attribute name="firstName" value="Jane"></attribute>
-        <attribute name="lastName" value="Doe"></attribute>
-        <attribute name="roles" value="Employee"></attribute>
-        <attribute name="roles" value="Manager"></attribute>
-        <attribute name="roles" value="Admin"></attribute>
+        <attribute name="firstName" value="Jane2"></attribute>
+        <attribute name="lastName" value="Doe2"></attribute>
+        <attribute name="roles" value="Employee2"></attribute>
+        <attribute name="roles" value="Manager2"></attribute>
+        <attribute name="roles" value="Admin2"></attribute>
     </attributes>
 </identity>
\ No newline at end of file
```

Reply:
Yes, that makes sense! I've updated it according to your suggestion, thanks